cli/lsp/testing/execution.rs

```diff
@@ -131,14 +122,14 @@ fn as_test_messages<S: AsRef<str>>(
 }
 
 #[derive(Debug, Clone, Default, PartialEq)]
-struct TestFilter {
-  maybe_include: Option<HashMap<String, TestDefinition>>,
-  maybe_exclude: Option<HashMap<String, TestDefinition>>,
+struct LspTestFilter {
```

Comment:
Arbitrary renaming, why?

Reply:
There is another `TestFilter` added in `cli/tools/test.rs`, it's used in this file